docker-compose.yml

```
@@ -13,3 +13,14 @@ services:
       - redis
+  dozzle:
+    container_name: dozzle
+    image: amir20/dozzle:latest
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+    ports:
+      - "8888:8080"
+    environment:
+      DOZZLE_USERNAME: ${DOZZLE_USERNAME}
+      DOZZLE_PASSWORD: ${DOZZLE_PASSWORD}
+    restart: always
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)